config/secrets.yml

```diff
@@ -95,6 +96,7 @@ test:
   vapid_private_key: <%= ENV['VAPID_PRIVATE_KEY'].presence || "vapid_private_key" %>
   login_token_time_limit: <%= ENV.fetch('LOGIN_TOKEN_TIME_LIMIT', '4320').to_i.abs.minutes %>
   submission_report_poll_time: <%= ENV.fetch('SUBMISSION_REPORT_POLL_TIME', 30)&.to_i %>
+  inactive_submission_review_allowed_time: <%= ENV.fetch('INACTIVE_SUBMISSION_REVIEW_ALLOWED_TIME', 30)&.to_i %>
```

Comment:
I have a feeling this should default to `0` when the variable is not set. i.e allow reviewing submissions by default.

Thinking further, this made me realize why even block reviewing pending submissions  though the access of student has ended. If we think about it the submission would have well been made when the student had access. If there is an operational reason to not consider the submission, the coach can always consider rejecting the submissions.

So, I feel all pending submissions should be gradable irrespective of student access. The coach anyway has a notice as well in the review page indicating end of access. Thoughts @harigopal ?

Reply:
I think we should have `INACTIVE_SUBMISSION_REVIEW_ALLOWED_DAYS` which defaults to `0`. When set to zero, it should preserve the current behaviour of blocking review immediately after a student's access has ended.